src/shrutiDJun20Assignments/PalindromeString1.java

```diff
@@ -0,0 +1,23 @@
+package shrutiDJun20Assignments;
+
+public class PalindromeString1 {
+
+	void palindromeStringCheck(String input) {
+		String rev = "";
+		for (int index = input.length()-1; index >= 0; index--) {
+			rev = rev + input.charAt(index);
+		}
+		if (input.equalsIgnoreCase(rev))
```

Comment:
It has to be equals not equalsIgnoreCase.

Reply:
changes done